src/utils/format-image-resizer-src/index.test.js

```diff
@@ -0,0 +1,57 @@
+import formatSrcset from ".";
```

Comment:
Should rename `formatSrcset` to `formatSrc` here for consistency.

Reply:
Got this one in your suggested commit. Thanks!